src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -543,11 +541,15 @@ def _forward(self, model_inputs, return_timestamps=False, generate_kwargs=None):
                     f"`input_features` or `input_values` key, but only has {model_inputs.keys()}"
                 )
 
-            # we need to pass `processed.get("attention_mask")` here since audio encoder
```

Comment:
We weren't doing anything special here, just pop'ing the attention mask from the inputs

=> I've moved the attention mask to the top of this method so that we only compute it once